src/TMainConsole.cpp

```diff
@@ -1442,10 +1442,11 @@ void TMainConsole::showStatistics()
         return;
     }
 
-    QString header = tr("+--------------------------------------------------------------+\n"
```

Comment:
:-1: Taking out the last line-feed out of a multi-line text is NOT helpful IMNSHO - it is not, as far as I can see, making things easier for translators and indeed they may wonder why *doesn't* the **last** line have the same form as the others.

Reply:
Can't follow your assumptions. Whitespace at edges has been troublesome for translators time and again. Also I found multiline text with no closing linefeeds in your own text as well, e.g. dlgRoomSymbols.cpp here, so can't be too bad:
```cpp
 instructions = tr("The only used symbol is \"%1\" in one or\n"
                              "more of the selected %n room(s), delete this to\n"
                              "clear it from all selected rooms or replace\n"
                              "with a new symbol to use for all the rooms:",
                              // Intentional comment to separate arguments!
                              "This is for when applying a new room symbol to one or more rooms "
                              "and some have the SAME symbol (others may have none) at present, "
                              "%n is the total number of rooms involved and is at least two. "
                              "Use line feeds to format text into a reasonable rectangle.",
                              mpRooms.size()).arg(mpSymbols.keys().first());
```

Reply:
That is because that is plain text in a `QDialog` and those line-feeds are required for all but the last line (and having one of the last line will IIRC produce an unwanted blank line when it is shown). For the text I am commenting about here, it is sent to the `(void) TConsole::print(const QString& msg, const QColor fgColor, const QColor bgColor)` method which then forwards the multi-line text to the appropriate `TBuffer` where it is appended and then "wrapped" - and the ending line-feed may or may not be required - I haven't determined it - to ensure the next line does start on the next line.